rg_instructor_analytics/static/rg_instructor_analytics/js/instructor_analytics.js

```diff
@@ -1,49 +1,260 @@
+/**
+ * module for instructor analytics tab
+ */
 (function () {
     'use strict';
     var CSS_INSTRUCTOR_CONTENT = 'instructor-dashboard-content-2';
 
     var tabList = [];
 
+    /**
+     * Abstract class for tabs in instructor analytics tab
```

Comment:
Does it really a class? It looks to be a function, doesn't it?

Reply:
It is class implementation, that compatible with pre ECMAScript 2015 standard

Reply:
👍